packages/augur-sdk/src/state/create-api.ts

```diff
@@ -208,6 +209,16 @@ export async function startServerFromClient(config: SDKConfiguration, client?: A
 export async function startServer(config: SDKConfiguration, account?: string): Promise<API> {
   const { api, sync } = await createServer(config, undefined, account);
 
+  api.augur.on(SubscriptionEventName.WarpSyncHashUpdated, async ({hash}) => {
```

Comment:
@pgebheim This event isn't firing. Also tried subscribing to NewBlock events without any luck. Any clue why the events wouldn't be pumping on this side?